host-controller/src/main/java/org/jboss/as/host/controller/logging/HostControllerLogger.java

```diff
@@ -1464,6 +1464,9 @@ void noDomainControllerConfigurationProvidedForAdminOnly(String policyAttribute,
     @Message(id = 217, value = "Security realms are no longer supported, please migrate references to them from the configuration.")
     XMLStreamException securityRealmReferencesUnsupported();
 
+    @Message(id = 218, value = "No server installation has been prepared.")
+    OperationFailedException noServerInstallationPrepared();
```

Comment:
@yersan Yes. My point though is less about giving more info and more about finding something to use instead of 'server' here, where we are talking about an installation that is running an HC.  For example, in the ServerLogger variant of this I think what you have is fine; using ProductConfig.getName() there instead of hard coded 'server' would be ok but just saying 'server' there is fine too. Here I can't think of a good word to hard code instead of 'server'.